batch-models/src/main/scala/org/ekstep/analytics/util/FileUtil.scala

```diff
@@ -7,6 +7,8 @@ import org.ekstep.analytics.framework.Level.INFO
 import org.ekstep.analytics.framework.util.JobLogger
 import org.sunbird.cloud.storage.conf.AppConf
 import org.sunbird.cloud.storage.factory.{StorageConfig, StorageServiceFactory}
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.SparkContext
```

Comment:
Why did this file change? Can you revert the changes?

Reply:
Done